README.md

```diff
@@ -223,31 +228,38 @@ to be followed. Once it is done, and after `mvn install`, the pipeline can be
 run using a command like:
 
 ```
-java -cp batch/target/fhir-batch-etl-bundled-0.1.0-SNAPSHOT.jar \
-  org.openmrs.analytics.FhirEtl --serverUrl=http://localhost:9018 \
-  --jsessionId=2950DA4C142EC44145978C02EDA0F311 \
-  --searchList=Patient,Encounter,Observation --batchSize=20 \
- --targetParallelism=20 --gcpFhirStore=GCP_FHIR_STORE`
+$ java -cp batch/target/fhir-batch-etl-bundled-0.1.0-SNAPSHOT.jar \
+    org.openmrs.analytics.FhirEtl --serverUrl=http://localhost:9018 \
+    --searchList=Patient,Encounter,Observation --batchSize=20 \
+   --targetParallelism=20 --gcpFhirStore=GCP_FHIR_STORE`
 ```
 The `searchList` argument accepts a comma separated list of FHIR search URLs.
 For example, one can use `Patient?given=Susan` to extract only Patient resources
-that meet the `given=Susan` criteria.
+that meet the `given=Susan` criteria. You can also dump the output into Parquet
+files using the `--outputParquetBase` flag.
 
+If you prefer not to use a bundled jar (e.g., during development in an IDE) you
+can use the Maven exec plugin:
+```
+$ mvn exec:java -pl batch -Dexec.mainClass=org.openmrs.analytics.FhirEtl \
+    "-Dexec.args=--serverUrl=http://localhost:9020  --searchList=Observation \
+    --batchSize=20 --targetParallelism=20 --outputParquetBase=tmp/TEST/" 
+```
```

Comment:
We might need in the future to have the ability to sync in both modes i.e:  local (parquet), cloud (gcp) or both. Is this something to parameterize? I assume if you pass --outputParquetBase it will automatically sync parquet while passing  --gcpFhirStore automatically syncs to GCP. What happens if you pass both?

Reply:
thanks @bashir2